src/java.desktop/share/classes/javax/swing/plaf/metal/MetalToolTipUI.java

```diff
@@ -147,80 +144,8 @@ public void paint(Graphics g, JComponent c) {
         }
     }
 
-    private int calcAccelSpacing(JComponent c, FontMetrics fm, String accel) {
-        return accel.isEmpty()
-               ? 0
-               : padSpaceBetweenStrings +
-                 SwingUtilities2.stringWidth(c, fm, accel);
-    }
-
     public Dimension getPreferredSize(JComponent c) {
         Dimension d = super.getPreferredSize(c);
-
-        String key = getAcceleratorString((JToolTip)c);
-        if (!key.isEmpty()) {
-            d.width += calcAccelSpacing(c, c.getFontMetrics(smallFont), key);
-        }
         return d;
     }
-
-    /**
-     * If the accelerator is hidden, the method returns {@code true},
-     * otherwise, returns {@code false}.
-     *
-     * @return {@code true} if the accelerator is hidden.
-     */
-    protected boolean isAcceleratorHidden() {
-        Boolean b = (Boolean)UIManager.get("ToolTip.hideAccelerator");
-        return b != null && b.booleanValue();
-    }
-
-    private String getAcceleratorString(JToolTip tip) {
-        this.tip = tip;
-
-        String retValue = getAcceleratorString();
-
-        this.tip = null;
-        return retValue;
-    }
-
-    /**
-     * Returns the accelerator string.
-     *
-     * @return the accelerator string.
-     */
-    // NOTE: This requires the tip field to be set before this is invoked.
-    // As MetalToolTipUI is shared between all JToolTips the tip field is
-    // set appropriately before this is invoked. Unfortunately this means
-    // that subclasses that randomly invoke this method will see varying
-    // results. If this becomes an issue, MetalToolTipUI should no longer be
-    // shared.
-    @SuppressWarnings("deprecation")
-    public String getAcceleratorString() {
```

Comment:
Although it is being placed in its super class so it can still be accessed
but am still not sure if you can remove this public method from this public class without notifying via @deprecated or @forRemoval tag

Reply:
I guess the deprecation was due to `getKeyModifiersText()`, which has been replaced with `getModifiersExText()`.